providers/shared/components/externalservice/state.ftl

```diff
@@ -64,16 +64,32 @@
     [#local core = occurrence.Core]
     [#local solution = occurrence.Configuration.Solution]
 
+    [#local cidrs = getGroupCIDRs(solution.IPAddressGroups, true, occurrence)]
+    [#local hostIPs = []]
+    [#list cidrs as cidr ]
+        [#local hostIPs = combineEntities(hosts, getHostsFromNetwork(cidr), UNIQUE_COMBINE_BEHAVIOUR) ]
+    [/#list]
+
+    [#local port = ports[solution.Port]]
+
+    [#local parentAttributes = parent.State.Attributes ]
+
     [#assign componentState =
         {
             "Resources" : {
                 "externalEndpoint" : {
                     "Id" : formatId(SHARED_EXTERNAL_RESOURCE_TYPE, core.Id),
                     "Name" : core.FullName,
-                    "Type" : SHARED_EXTERNAL_RESOURCE_TYPE
+                    "Type" : SHARED_EXTERNAL_RESOURCE_TYPE,
```

Comment:
Think the type is wrong - ^C^V?

Reply:
I left it as an external resource type intentionally, it didn’t feel like it needed a distinct resource when it’s not actually used

Reply:
hmm - not sure I'm comfortable with this but I'll commit for mow